internal/impl/sql/integration_test.go

```diff
@@ -712,21 +724,20 @@ func postgresIntegration(t *testing.T) {
 }
 
 func mySQLIntegration(t *testing.T) {
-	if runtime.GOOS == "darwin" {
-		t.Skip("skipping test on macos")
-	}
-
 	t.Parallel()
 
 	pool, err := dockertest.NewPool("")
 	if err != nil {
 		t.Skipf("Could not connect to docker: %s", err)
 	}
-	pool.MaxWait = 30 * time.Second
+	pool.MaxWait = 3 * time.Minute
 
 	resource, err := pool.RunWithOptions(&dockertest.RunOptions{
 		Repository:   "mysql",
 		ExposedPorts: []string{"3306/tcp"},
+		Cmd: []string{
+			"--sql_mode=ANSI_QUOTES",
```

Comment:
This forces MySQL to allow double quotes around column names instead of Backticks. Backticks are not compatible with all the other DB engines.